### PR TITLE
Proper lobby art system

### PR DESCRIPTION
**Pull request by XElectricX**
## About The Pull Request

Was just going to be looping through different lobby art images but it necessitated a lot of changes.

- Lobby art fades into the next image
- How long the art stays up and how long the fading is can be changed via config (default 30 and 5 seconds respectively)
- Split up each piece of art into it's own file; had to give them names so made some up (if you are the artist of one of these images or know the title, please comment it)
- The system automatically looks for any images in the designated directory to pick an image to use (said directory can also be changed via the config)
- This means any future artist can also PR a new image quite easily, just drop a new .dmi file in the directory (the sprite name MUST be blank)

A map edit was made since the lobby art was actually a `turf`, which caused issues (and is a dumb way to handle it). The rest of the edits were StrongDMM doing it's sanitization. This PR **requires** #15848 to work, otherwise the players won't see anything.

## Why It's Good For The Game

We love lobby art. Let us see the rest.

## Changelog
:cl:
imageadd: Split the lobby art file into their own individual files.
imagedel: Deleted old lobby art file.
refactor: Lobby art system exists now instead of just being an icon_state assignment on a turf.
config: Lobby art directory, duration, and fade time can be set in config.
/:cl:

### CODE REVIEW

**Comment by XElectricX** [comment]
I think the icon bot might have broken from checking too many files.

File: code/game/objects/effects/lobby_art.dm [hunk 1]
```diff
@@ -0,0 +1,46 @@
+GLOBAL_LIST_EMPTY(lobby_art_files)
+
+/obj/effect/lobby_art
```
Comment (TiviPlus) [inline code/game/objects/effects/lobby_art.dm:3]: ?? why

File: code/game/objects/effects/lobby_art.dm [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+/obj/effect/lobby_art/LateInitialize()
+	. = ..()
+	if(length(GLOB.lobby_art_files) > 1)	//Don't bother changing the art if there's only one
+		addtimer(CALLBACK(src, PROC_REF(transition_art)), CONFIG_GET(number/lobby_art_duration) SECONDS)
+
+///Create a new lobby art object and delete this one
+/obj/effect/lobby_art/proc/transition_art()
+	new /obj/effect/lobby_art(loc, icon_state)
+	layer -= 0.1	//I could use initial() but I don't see an instance to where anything should be changing the layer of it (minus this)
+	QDEL_IN(src, CONFIG_GET(number/lobby_art_fade_duration) SECONDS)
```
Comment (TiviPlus) [inline code/game/objects/effects/lobby_art.dm:28]: Im denying the fade, we already have animated lobby UIs that swap and I dislike how it looks, people should be able to look at one

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/lobby_art.dm (hunk 1 above)
Comment (XElectricX) [reply code/game/objects/effects/lobby_art.dm:3]: Why what? `effect` types are good for abstract things that shouldn't be interacted with.

File: code/game/objects/effects/lobby_art.dm (hunk 2 above)
Comment (XElectricX) [reply code/game/objects/effects/lobby_art.dm:28]: That is quite literally just your opinion. I think it looks great. The animated bit is also silly, they still play out completely within 30 seconds. I could also just isolate the animated ones so the art doesn't transition if an animated one is selected roundstart if it really bugs you.

TM it and if the majority of people hate it then I can just close it. 🤷

File: code/game/objects/effects/lobby_art.dm (hunk 1 above)
Comment (TiviPlus) [reply code/game/objects/effects/lobby_art.dm:3]: It was a turf earlier though?

File: code/game/objects/effects/lobby_art.dm (hunk 2 above)
Comment (XElectricX) [reply code/game/objects/effects/lobby_art.dm:28]: Who died and made you the aesthetics police???

File: code/game/objects/effects/lobby_art.dm (hunk 1 above)
Comment (XElectricX) [reply code/game/objects/effects/lobby_art.dm:3]: And now it's not.

**Comment by TiviPlus** [comment]
If you dont wanna change it ok then

**Pull request closed without merging**